src/codeFly/frontEnd/about.html

```diff
@@ -0,0 +1,175 @@
+                    <h3 align=center>Our Compiler Team</h3>
+                </div>
+            </div>
+            <div class="col-md-9 order-md-1">
+                <div class="card card-body">
+                    <h2>Team introduction</h2>
+                    <p>Place holder</p>
+                    <p><a href="#">More &raquo;</a></p>    
+                </div>
+            </div>
+            <!--Out-of-date 
+            <div class="col-xs-12 col-sm-3 col-sm-push-9">
+                <p style="padding:20px;"></p>
+                <h3 align=center>Meet our Culinary Specialists</h3>
+            </div>
+            <div class="col-xs-12 col-sm-9 col-sm-pull-3">
+                <h2>Alberto Somayya</h2>
+                <h4>Executive Chef</h4>
+                <p>Award winning three-star Michelin chef with wide International experience having worked closely with whos-who in the culinary world, he specializes in creating mouthwatering Indo-Italian fusion experiences. </p>
+                <p><a href="#">More &raquo;</a></p>
+            </div>
+            -->
+
+        </div>
+    </div>
+
+    <footer>
+        <div class="container">
+            <div class="row">             
+                <div class="col-xs-5 col-xs-offset-1 col-sm-2 col-sm-offset-1">
+                    <h5>Links</h5>
+                    <ul class="list-unstyled">
+                        <li><a href="#">Home</a></li>
+                        <li><a href="#">About</a></li>
+                        <li><a href="#">Menu</a></li>
+                        <li><a href="#">Contact</a></li>
+                    </ul>
+                </div>
+                <div class="col-xs-12 col-sm-5">
+                    <h5>Our Address</h5>
+                    <address>
+		              #xxx, xx Road<br>
+		              Syracuse University<br>
+		              New York<br>
+		              Tel.: +xxx<br>
+		              Fax: +xxx<br>
+		              Email: <a href="mailto:CodeFly@syracuse.edu">codefly@syracuse.edu</a>
+		           </address>
+                </div>
+                <div class="col-xs-12 col-sm-4">
+                    <div style="padding: 40px 10px;">
+                        <a href="http://google.com/+">Google+</a>
+                        <a href="http://www.facebook.com/profile.php?id=">Facebook</a>
+                        <a href="http://www.linkedin.com/in/">LinkedIn</a>
+                        <a href="http://twitter.com/">Twitter</a>
+                        <a href="http://youtube.com/">YouTube</a>
+                        <a href="mailto:">Mail</a>
+                    </div>
+                </div>
+                <div class="col-xs-12">
+                    <p style="padding:10px;"></p>
+                    <p align=center>© Copyright 2018 CodeFly</p>
+                </div>
+            </div>
+        </div>
+    </footer>
+    <!-- jQuery (necessary for Bootstrap's JavaScript plugins) -->
+    
```

Comment:
I have commented out all the scripts at the end of file. It seems that they do not contribute to the rendering of your HTML